### DISCUSSION TITLE
looking for maintainers

### DISCUSSION THREAD

**Discussion by raybellwaves** [aws-azure-login/aws-azure-login#299] [General]
Given that someone e-mailed me personally to ask about the status of this repo I thought I would invite other maintainers.

For context, I ended up as a maintainer as this repo once sat in [@DTN-Public ](https://github.com/DTN-Public). The person who took ownership of the repo from https://github.com/sportradar left the company and because i'm the only person active on https://github.com/DTN-Public I inherited maintenance. I moved the repo to https://github.com/aws-azure-login to encourage public ownership.

I have little knowledge of this repo and have no interest in maintaining it. If anyone is actively contributing to this repo I would be more than happy to have them maintain it. Here are the current maintainers:

![Screen Shot 2023-08-02 at 11 47 27 AM](https://github.com/aws-azure-login/aws-azure-login/assets/17162724/ed759377-bdef-49b8-bd9f-929f3a007f1f)

**Comment by pzrq** [comment]
Obligatory maintainer burnout is a thing post, though perhaps someone from one of these two megacorp[orations] might see this: https://www.jeffgeerling.com/blog/2022/burden-open-source-maintainer

**Comment by keanu-001** [comment]
Hi @raybellwaves. I am interested in becoming a maintainer. I work full-time as a software engineer but use this tool regularly and would like to contribute

**Reply by raybellwaves** [reply]
done

**Comment by robrich** [comment]
Hi @raybellwaves. I too am interested in becoming a maintainer. I use this tool at my employer currently and would like to see a new release with AWS v3.

**Reply by raybellwaves** [reply]
Better to ask @Naicigam28
